pyproject.toml

```diff
@@ -1,20 +1,122 @@
+[project]
+name = "rod"
+dynamic = ["version"]
+requires-python = ">= 3.10"
+description = "The ultimate Python tool for RObot Descriptions processing."
+authors = [
+    { name = "Diego Ferigo", email = "dgferigo@gmail.com" },
```

Comment:
There is always a fine line between who is author and who not, but I guess that Filippo and Lorenzo having contributed ~36 and ~20 commits each perhaps can be listed as authors as well? fyi @diegoferigo

Reply:
To me the author is who created the project and developed most of its core, contributors are those that contributed with minor features, and maintainers who is actively taking care of chores and possibly contributing with minor features. What do you think?

Reply:
To me the author is who created the project and developed most of its core, contributors are those that contributed with minor features, and maintainers those that are actively taking care of chores and possibly contributing with minor features (but not necessarily). What do you think?

Feel free to adjust the README accordingly (note that github already shown on the right panel the contributors). PyPI metadata has only authors/maintainers. Contributors generally can be gathered from github. Usually for external people, it's more important reaching out through email only the original author / current maintainer.

Reply:
Ok for me, I typically always consider the maintainers team a subset of the authors teams, but that is just personal preference, but I do not strongly feel about this.

Reply:
Ok, just to clarify, the role of a maintainer -as I see it- is more loose. It can either be one of the authors (in case of major contributions), one of the contributors (in case of minor contributions), or just a maintainer (in case of just chores).